envs/6.0

```diff
@@ -1,8 +1,8 @@
 COMPOSE_PROJECT_NAME=polymesh
 
 CHAIN_IMAGE=polymeshassociation/polymesh:99dc6cc1b2-develop-debian
```

Comment:
`CHAIN_IMAGE=polymeshassociation/polymesh:99dc6cc1b2-develop-debian` can be
`CHAIN_IMAGE=polymeshassociation/polymesh:6.0.0-staging-debian` now.